lib/stats.js

```diff
@@ -86,15 +86,15 @@ module.exports = {
    * @param {object} stats
    * @returns {object|null}
    */
-  getTheMostCommon(stats) {
+  getTheMostCommon(components) {
     let max = null;
 
-    for (const key of Object.keys(stats)) {
-      if (!max || max.count < stats[key].count) {
-        max = stats[key];
+    for (const key of Object.keys(components)) {
```

Comment:
That doesn't work. It always returns the first thing in components. In the project I work in for daily basis I see this:

```
The most used component: admin-authentication-form (1 occurrence)
```

Instead of:
```
The most used component: ui-form-button (176 occurrences)
```

The code should look like this:
```suggestion
    for (const key of Object.keys(components)) {
      if (!max || max.stats.count < components[key].stats.count) {
        max = components[key];
      }
    }

    return max;
```

Reply:
I added some tests for the stats to make sure this is working